index.bs

```diff
@@ -50,19 +51,51 @@ This provides similar functionality as earlier drafts of the
 
 ## Concepts ## {#concepts}
 
-A <dfn>entry</dfn> is either a [=file entry=] or a [=directory entry=].
+An <dfn>entry</dfn> is either a [=file entry=] or a [=directory entry=].
 
 Each [=/entry=] has an associated <dfn for=entry>name</dfn>.
 
 A <dfn lt="file|file entry">file entry</dfn> additionally consists of <dfn for="file entry">binary
 data</dfn> and a <dfn for="file entry">modification timestamp</dfn>.
 
 A <dfn lt="directory|directory entry">directory entry</dfn> additionally consists of a [=/set=] of
-<dfn for="directory entry">entries</dfn>. Each member is either a [=file=] or a [=directory=].
+<dfn for="directory entry">children</dfn>, which are themselves [=/entries=]. Each member is either a [=file=] or a [=directory=].
 
-Issue: TODO: Explain how entries map to files on disk (multiple entries can map to the same file or
+[=/Entries=] can (but don't have to) be backed by files on the systems native file system,
+as such it is possible for the [=binary data=], [=modification timestamp=],
+and [=children=] of entries to be modified by applications outside of this specification.
+Exactly how external changes are reflected in the data structures defined by this specification,
+as well as how changes made to the data structures defined here are reflected externally
+is left up to individual user-agent implementations.
+
+An [=/entry=] |a| is <dfn for="entry">the same as</dfn> an [=/entry=] |b| if |a| is equal to |b|, or
+if |a| and |b| are backed by the same file or directory on the native file system.
+
+Issue: TODO: Explain better how entries map to files on disk (multiple entries can map to the same file or
 directory on disk but doesn't have to map to any file on disk).
```

Comment:
(not introduced in this commit)

"but an entry doesn't have to map to..."?

The previous clause's subject is "multiple entries" (plural). This sentence probably needs a different subject, or a singular/plural agreement fix (doesn't -> don't).